brainstormforce/ultimate-addons-for-gutenberg
# feat: Inject UAG Google Fonts in Astra & Load only one common Google Fonts File for both.

### Description
feat: Inject UAG Google Fonts in Astra & Load only one common Google Fonts File for both.

### Types of changes
New feature (non-breaking change which adds functionality)

### How has this been tested?
- Test UAG & Astra Google Fonts Working to see if all fonts apply.
- Test if a common file is loaded for both.
- Check by using Astra's Load Local Google Fonts option if everything is working fine.
- Check by switching to a different theme.

### Checklist:
